src/common/options/global.yaml.in

```diff
@@ -5365,6 +5365,30 @@ options:
   desc: Enable health indication when spurious read errors are observed by OSD
   default: true
   with_legacy: true
+- name: bluestore_slow_ops_warn_lifetime
+  type: uint
+  level: advanced
+  desc: Duration of slow op warning of bluestore should persist since last occurence
+  default: 86400
+  with_legacy: true
+- name: bluestore_slow_ops_warn_threshold
+  type: uint
+  level: advanced
+  desc: A configurable number for slow ops warning to be appeared if number of occurence pass this threshold
+  default: 1
+  with_legacy: true
+- name: bdev_stalled_read_warn_lifetime
```

Comment:
`bdev_` related options should go somewhere close to bdev other options.
Options should refer related options by `see_also:` terms.